fuse_variables/test/launch_tests/test_load_device_id.cpp

```diff
@@ -33,20 +33,51 @@
  */
 #include <fuse_core/uuid.h>
 #include <fuse_variables/stamped.h>
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 
 #include <gtest/gtest.h>
 
 #include <stdexcept>
 #include <string>
+#include <thread>
 
+class TestLoadDeviceId : public ::testing::Test
+{
+public:
+  void SetUp() override
+  {
+    exec_ = std::make_shared<rclcpp::executors::SingleThreadedExecutor>();
+
+    spinning_ = true;
```

Comment:
I don't think spinning is needed here, just `exec_->cancel()` to stop the thread.